core/readme.md

```diff
@@ -1,86 +1,186 @@
-4. Someone is picked to write a CAP for a given proposal. They then:
-    * Fork the repository by clicking "Fork" in the top right.
-    * Write the CAP in their fork of the repository. There is a [template CAP here](../cap-template.md). The first PR should be a first draft of the CAP. It must follow the template.
-    * Add a link to the CAP proposal in this document, linking to the appropriate `/core/cap-XXXX.md`
-    * If your CAP requires images or other supporting files, they should be included in a subdirectory of the `contents` folder for that CAP as follows: `contents/cap-X` (for CAP **X**). Links should be relative, for example a link to an image from CAP-X would be `../contents/cap-X/image.png`.
-    * Drafts are numbered by the author but subject to change when made final
-    * CAP is written and set as `Draft`
-    * Submit a Pull Request to Stellar's [protocol repository](https://github.com/stellar/stellar-protocol).
-5. Buddy is assigned (Jon, Graydon, Jeremy, Johnny, Orbitlens) and will merge your PR if you properly followed the steps above.
-6. Discussion of the draft CAP will now take place on the mailing list. There should be some iteration between discussion and further PRs refining the CAP.
-7. If the CAP is not receiving enough support the Buddy will mark as rejected and move it to the archive. Otherwise the CAP is moved to `Pending`. 
-8. There is now one week for people to make final comments.
-9. After that the CAP is discussed at the next protocol meeting.
-10. Needs unanimous approval from (Nicolas, Jed, David) to become `Accepted` otherwise the CAP is `Rejected` or turned back to `Draft` with some comments.
-11. Implementer is decided and that person starts implementing.
-12. Once a CAP is implemented, a PR should be submitted to update its status to `Final`.
-
-
-
-**CAP Key Members**: Nicolas, Jed, David
-
-**CAP Buddies**: (Jon, Graydon, Jeremy, Johnny, Orbitlens)<BR>
-Buddies are responsible for moving a CAP along the process. They should make sure the draft is either `Accepted` or `Rejected` in a timely manner.
-
+These are the steps from [idea to deployment](https://www.youtube.com/watch?v=Otbml6WIQPo) on how
+to create a Core Advancement Proposal (CAP).
+
+### Pre-CAP (Initial Discussion)
+Introduce your idea on the [stellar-dev mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/stellar-dev).
+
+* Make sure to gather feedback and alternative ideas — it's useful before putting together a
+  formal draft!
+* Consider contacting experts in a particular area for feedback while you're hashing out the
+  details.
+
+### Creating a CAP Draft
+Draft a formal proposal using the [CAP Template](../cap-template.md), and submit a PR to this
+repository. You should make sure to adhere to the following:
+
+* Make sure to place the draft in the `core/drafts` folder.
+* Use the following format for the filename of your draft
+  `cap_{email_username}_{hyphen-separated-cap-title}.md`.
```

Comment:
Maybe github username instead of email? Email is a little longer / more awkward and may feel exposing to submitters.

Reply:
Yup, makes sense — see update for using the sha256 of the file to make it reasonably unique when folks are submitting a PR.